cmd/tester/cmd/ibc_transfer.go

```diff
@@ -0,0 +1,176 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/b-harvest/cosmos-module-stress-test/client"
+	"github.com/b-harvest/cosmos-module-stress-test/config"
+	"github.com/b-harvest/cosmos-module-stress-test/tx"
+	"github.com/b-harvest/cosmos-module-stress-test/wallet"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/applications/transfer/types"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagPacketTimeoutHeight    = "packet-timeout-height"
+	flagPacketTimeoutTimestamp = "packet-timeout-timestamp"
+	flagAbsoluteTimeouts       = "absolute-timeouts"
+)
+
+func IbctransferCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "transfer [src-port] [src-channel] [receiver] [amount] [round] [tx-num] [msg-num]",
+		Short:   "Transfer a fungible token through IBC",
+		Aliases: []string{"t"},
+		Args:    cobra.ExactArgs(7),
+		Long: `Transfer a fungible token through IBC.
+
+Example: $tester t transfer channel-0 cosmos1pacc0fr45hggcn8jrfhgnqf8vgyqna7r5sftql 10uatom 10 1 1
+
+[round]: how many rounds to run
+[tx-num]: how many transactions to be included in one round
+`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			logLvl, err := zerolog.ParseLevel(logLevel)
+			if err != nil {
+				return err
+			}
+
+			zerolog.SetGlobalLevel(logLvl)
+
+			switch logFormat {
+			case logLevelJSON:
+			case logLevelText:
+				// human-readable pretty logging is the default logging format
+				log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
+			default:
+				return fmt.Errorf("invalid logging format: %s", logFormat)
+			}
```

Comment:
This can be updated to use [SetLogger](https://github.com/b-harvest/cosmos-module-stress-test/blob/main/cmd/tester/cmd/root.go#L41).

```go
logLvl, err := zerolog.ParseLevel(logLevel)
if err != nil {
	return err
}

zerolog.SetGlobalLevel(logLvl)

switch logFormat {
case logLevelJSON:
case logLevelText:
	// human-readable pretty logging is the default logging format
	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
default:
	return fmt.Errorf("invalid logging format: %s", logFormat)
}
```